doc/doc-site/docs/overview/orders.md

```diff
@@ -215,297 +215,214 @@ Econia does not use a critical height of 1. The actual critical height is define
+Therefore, we want a tick size of 0.00001 to achieve the lot value granularity required to express a price of \$0.00732 for one lot.
+Let's now get that in subunit terms using the SDK:
 
 ```python
->>> tick_size_integer = int(lot_size_decimal * tick_size_decimal
-...                         * 10 ** decimals_quote)
->>> tick_size_integer
-1
+>>> tick_size_unit = "0.00001"
+>>> quote_decimals = 6
+>>> get_tick_size_integer(tick_size_unit, quote_decimals)
+10
 ```
 
-A decimal tick size of 0.001 `USDC` per `APT` would not be supported, however, because at the lowest possible price of 1 tick, the increment in total quote amount for each additional lot could not be represented as an integer multiple of subunits:
+It's easy to tell if this lot size and tick size combination is reasonable by checking the minimum and maximum price under these conditions in nominal (unit) terms.
+Note that the minimum price is also equivalent to price granularity--this tells us the granularity of value for a whole unit of `APT` as opposed to just one lot (equal here to 100000 subunits of `APT`).
+Now, using the SDK:
 
 ```python
->>> tick_size_decimal = dec('0.001')
->>> quote_increment_decimal = lot_size_decimal * tick_size_decimal
->>> (quote_increment_decimal * 10 ** decimals_quote).normalize()
-Decimal('0.1')
+>>> get_min_quote_per_base_nominal(lot_size_unit, tick_size_unit)
+0.01
+>>> get_max_quote_per_base_nominal(lot_size_unit, tick_size_unit)
+42949672.95
 ```
 
-Hence to check that a lot size/tick size combination is even possible for a market:
+It looks like our price granularity is one penny and our maximum price is "plenty" high.
+Since the current price of `APT` is \$7.32, this price granularity might be too coarse for higher-frequency traders.
+It's a good thing we checked!
+Fixing this is easy enough: we could increase our lot size which would make our size granularity more coarse than we desire, or we could decrease our tick size by one order of magnitude:
 
-1. Pick a decimal lot size.
+```python
+>>> tick_size_unit_new = "0.000001"
+>>> get_tick_size_integer(tick_size_unit_new, quote_decimals)
+1
+```
 
-   ```python
-   >>> lot_size_decimal = dec('0.1')
-   ```
+Great!
+That just so happens to be the smallest possible tick size in this case.
+Now let's plug this tick size back into the minimum and maximum price formulas to check our work:
 
-1. Assert that integer lot size corresponds to an integer multiple of base subunits.
+```python
+>>> get_min_quote_per_base_nominal(lot_size_unit, tick_size_unit_new)
+0.001
+>>> get_max_quote_per_base_nominal(lot_size_unit, tick_size_unit_new)
+4294967.295
+```
 
-   ```python
-   >>> to_check = lot_size_decimal * 10 ** decimals_base
-   >>> lot_size_integer = int(to_check)
-   >>> assert to_check >= 1 and to_check == lot_size_integer
-   ```
+Notice the maximum price has gone down by one order of magnitude, but it's still "plenty" high.
```

Comment:
I suggest no quotes around plenty, and maybe be more concrete (e.g. supports APT prices up to $4.2 M per APT)